### PR TITLE
add glossary proposal

### PR DESCRIPTION
**Pull request by xh3b4sd**
I think it would make sense to start writing down a little bit more about our terminologies. So we can be sure to talk about the same things. Clarifying the abbreviations also helps when writing in comments and the like.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,22 @@
...
 
 - [Naming Resources](azure/naming_resources.md)
 
 
 
+## Glossary
+
+- [Control Plane](glossary/control_plane.md)
+- [Installation](glossary/installation.md)
+- [Tenant Cluster](glossary/tenant_cluster.md)
```
Comment (kopiczko) [inline README.md]: I feel like the whole glossary should be rather a single page. You can maybe put it under `General` section. I can see over time becoming really long list of single sentence pages.

**Review by kopiczko — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (xh3b4sd) [reply README.md]: I found it quite cool to see the separate files and the list in the readme. I don't think this list will become very long. We are not going to have more than 10 GS specific terms to talk about I guess. Or, I hope. :D

Comment (rossf7) [reply README.md]: I'd also be fine with a single page. But no blocker if you'd prefer them to be separate.

**Review by rossf7 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (tuommaki) [reply README.md]: I'd also prefer single page for easier navigation (or more like lack of unneeded navigation), but no blocker for me as well.

**Review by tuommaki — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (xh3b4sd) [reply README.md]: When the majority thinks a single page is better, let's do this when the list gets longer, or feel free to provide a PR. Thank you for your feedback.

**Pull request merged**